### ISSUE TITLE
Unable to View Instructions for lambda compiled method when there are more than one with the same parameter

### ISSUE THREAD

**Issue by yyjdelete** [sailro/Reflexil#22]
Unable to View Instructions for lambda compiled method when there are more than one with the same parameter
Tested with Reflexil 1.9
`ConsoleApplication3.Program.<>c__DisplayClass0_0.<cc>b__2(object):void`
 can be view,
while `ConsoleApplication3.Program.<>c.<cc>b__0_0(object):void`/`ConsoleApplication3.Program.<>c.b__0_1(object):void` can't.

```csharp
using System;

namespace ConsoleApplication3
{
	class Program
	{
		public static void cc()
		{
			string a = "<a";
			Action<object> print = s=> Console.Out.WriteLine(s);
			Action<object> print1 = s=> Console.Out.WriteLine(s + "1");
			Action<object> printa = s=> Console.Out.WriteLine(s + a);

			print("aabbccdd");
			printa("aabbccdd");
		}
		static void Main(string[] args)
		{
			cc();
			Console.ReadKey();
		}
	}
}
```

**Comment by yyjdelete** [comment]
Only Failed for Reflector, 
Tested OK for reflexil.for.ILSpy.2.0-preview3.zip in https://github.com/sailro/Reflexil/issues/2

**Comment by sailro** [comment]
Interesting.

In fact ILSpy/Reflexil speak the same "dialect", using Mono.Cecil lib.
And Reflector and Reflexil have a different code model so that's why you can see differences.

I will check that.
Thanks

**Comment by sailro** [comment]
Fixed in 8fe35b1992828eb58aeaaf05f5573dca3fee87c5

Thanks
